terraform/testcases/statsd_mock/otconfig.tpl

```diff
@@ -0,0 +1,15 @@
+receivers:
+  statsd:
+    endpoint: 0.0.0.0:${udp_port}
+    aggregation_interval: 60s
+exporters:
+  logging:
+    loglevel: debug
+  otlphttp:
+    metrics_endpoint: "https://${mock_endpoint}"
+    insecure: true
+service:
+  pipelines:
+    metrics:
+      receivers: [statsd]
+      exporters: [otlphttp, logging]
```

Comment:
could you remove logging because this config will also be used in performance test and logging would impact the result.

Reply:
Sure, will remove. Sure, I can include the changes for performance test later.